Packs/Slack/Integrations/SlackV3/SlackV3.yml

```diff
@@ -0,0 +1,550 @@
+category: Messaging
+commonfields:
+  id: SlackV3
+  version: -1
+configuration:
+- display: ""
+  displaypassword: Slack API access token
+  name: access_token
+  type: 9
+  required: false
+  hiddenusername: true
+- display: ""
+  displaypassword: Slack API bot token
+  name: bot_token
+  type: 9
```

Comment:
I'm not sure. I'm personally not a fan of the implementation here since it does not look good in server versions lower than 6.1. I'd rather just keep the parameters handled as encrypted fields.